run-upgrade-tests.sh

```diff
@@ -61,6 +71,8 @@ pifpaf_stop(){
 cleanup(){
     pifpaf_stop
     rm -rf $GNOCCHI_DATA
+    indexer_stop
+    [ "$STORAGE_DAEMON" == "ceph" ] && storage_stop
```

Comment:
just need to add `|| true` so it doesn't return non zero code.

Reply:
Thanks!